tests/cli/commands/test_dag_command.py

```diff
@@ -470,6 +471,28 @@ def test_cli_report(self):
         assert "airflow/example_dags/example_complex.py" in out
         assert "example_complex" in out
 
+    @conf_vars({("core", "load_examples"): "true"})
+    def test_cli_get_dag_details(self):
+        args = self.parser.parse_args(["dags", "details", "example_complex", "--output", "yaml"])
+        dag_id = "example_complex"
+        with contextlib.redirect_stdout(io.StringIO()) as temp_stdout:
+            dag_command.dag_details(args)
+            out = temp_stdout.getvalue()
+
+        with create_session() as session:
+            dag = session.query(DagModel).filter(DagModel.dag_id == dag_id).one_or_none()
+            dag_detail_fields = dag_schema.dump(dag)
```

Comment:
you can access sqlalchemy table columns by this simple way without querying the database
```suggestion
        dag_detail_fields = [column.name for column in DagModel.__table__.columns]
```

Reply:
Yeah this works too, though had to skip one field. Changes done.

Reply:
I found a solution to avoid the skip condition by using:
```python
dag_detail_fields = DAGSchema().fields.keys()
```